src/components/potentialTable.js

```diff
@@ -1414,27 +1560,12 @@ export default function PotentialTable() {
       >
         <FormGroup>
           <FormControlLabel
-            control={
-              <Switch
-                color="primary"
-                checked={switchChecked}
-                onChange={toggleSwitch}
-              />
-            }
+            control={<Switch checked={switchChecked} onChange={toggleSwitch} />}
```

Comment:
Consider changing toggleswitch on on by default

Reply:
It's now on by default.
`const [switchChecked, setSwitchChecked] = useState(true);`